### DISCUSSION TITLE
Removing of legacy dependencies from "@project-sunbird/sunbird-collection-editor"

### DISCUSSION THREAD

**Discussion by abhishekpnt** [Sunbird-Knowlg/Community#119] [General Discussions]
Hi team,

Since we are upgrading Angular version of the Sunbird-Ed portal, we are doing legacy dependency cleanup activity. 
    But it was found that "**@project-sunbird/sunbird-collection-editor**" has some legacy dependencies like **ngx-infinite-scroll, angular2-uuid,common-form-elements-web-v9,ng2-cache-service**   which are being removed from the portal. 
    Since these dependencies are being added as the "dependency" in "@project-sunbird/sunbird-collection-editor" , when removing these dependencies from portal , we are getting  the build issues in the portal. 

So please remove/update these legacy dependencies from the above mentioned package and give us the latest version  without the above list of legacy dependencies. 

Also please try to remove  "**@project-sunbird/sunbird-pdf-player-v9**","**@project-sunbird/sunbird-quml-player-v9**","**@project-sunbird/sunbird-video-player-v9**" from package.json since these modules will be used as a web component in the portal

Please refer the ticket: https://project-sunbird.atlassian.net/browse/ED-1277
@pallakartheekreddy can you please help us on this?

CC:  @vinukumar-vs @Krishnaj20  @rajeevsathish @rajeshkumaravel @princegupta1131

**Comment by pallakartheekreddy** [comment]
@abhishekpnt @vinukumar-vs We have removed the above packages as part of release-5.4.0 and published a new tag for collection editor. Please use release-5.4.0 release notes for the latest tag.  Please refer below links for more details
https://project-sunbird.atlassian.net/browse/KN-831
https://knowlg.sunbird.org/use/release-notes/release-5.4.0-latest
@Krishnaj20

**Reply by abhishekpnt** [reply]
Hi @pallakartheekreddy In version 5.4.0 , i see that ngx-infinite-scroll and common-form-elements-web-v9 dependencies are not removed yet

Please refer:
https://github.com/Sunbird-Knowlg/sunbird-collection-editor/blob/release-5.4.0/package.json#L54
https://github.com/Sunbird-Knowlg/sunbird-collection-editor/blob/release-5.4.0/package.json#L59

So getting the error 
![image](https://user-images.githubusercontent.com/116337484/229782077-63a79b1c-8770-43bd-a683-b44d48cb189f.png)

CC: @vinukumar-vs @rajeevsathish

**Reply by pallakartheekreddy** [reply]
@abhishekpnt Yes, we cross-checked we haven't removed `ngx-infinite-scroll` as it is used by `sunbird-resource-library` which is contributed by Samagra. We will plan to remove that in an upcoming release, and we will integrate players also as a web components inside the editor.  
@vinukumar-vs @Krishnaj20
